transports/webrtc/src/tokio/transport.rs

```diff
@@ -118,7 +118,15 @@ impl libp2p_core::Transport for Transport {
         }
     }
 
-    fn dial(&mut self, addr: Multiaddr) -> Result<Self::Dial, TransportError<Self::Error>> {
+    fn dial(
+        &mut self,
+        addr: Multiaddr,
+        _: DialOpts,
+    ) -> Result<Self::Dial, TransportError<Self::Error>> {
+        // TODO: As the listener of a WebRTC hole punch, we need to send a random UDP packet to the
+        // `addr`. See DCUtR specification below.
+        //
+        // https://github.com/libp2p/specs/blob/master/relay/DCUtR.md#the-protocol
```

Comment:
It is not clear that this TODO applies only to `DialOpts.role.is_listener()`.

Reply:
I hope I changed that satisfactory.